README.md

```diff
@@ -282,6 +282,17 @@ use. For example, if you press `w` the status bar will change from the modal
 command icon `[=]` to `[w]` (the window command). If you now further press `s`,
 it will update to `[ws]` to signify the split window command sequence and so on.
 
+### Set custom status bar modal mode icon
+The option `@modal-cmd-icon` can be used to specify any string, which will
+be the left status bar icon to indicate that modal mode (key tables) are in
+use. Default `[=]`.
+
+```
+set -g @modal-cmd-icon "..."
+```
+
+If you have Nerd Fonts installed, you can use an icon like the hourglass `"󰔟"`:
```

Comment:
I think that most people don't have Nerd Fonts installed, and can thus not render the font (just like Github's Markdown previewer). We should use some other sample icon that might be more readily available.